docs/src/remote-wallet/README.md

```diff
@@ -32,8 +33,7 @@ usb://<MANUFACTURER>[/<PRODUCT>[/<WALLET_KEY>]][?key=<DERIVATION_PATH>]
 `PRODUCT` is optional and defaults to an auto-detected value. When using a Ledger
 Nano S, for example, it defaults to "nano-s" without quotes.
 
-`WALLET_KEY` is used to disambiguate multiple devices. Each device has a unique
-master key and from that key derives a separate unique key per app.
+`WALLET_KEY` is a globally unique key used to disambiguate multiple devices.
```

Comment:
The use of "key" here is a bit ambiguous given the context. Perhaps "id" would do the job?

Reply:
maybe go the other way and explain how that's a perfectly valid public key?

Reply:
Ah, could do yeah.  From the "globally unique" bit I assumed this was some vendor-assigned ID.

Presumably one could initialize two ledger devices with the same seed phrase and make a weird globe?